meilisearch-http/src/routes/setting.rs

```diff
@@ -91,6 +91,19 @@ async fn get_all(
 
     let schema = index.main.schema(&reader)?;
 
+    let attributes_for_faceting = match (&schema, &index.main.attributes_for_faceting(&reader)?) {
+        (Some(schema), Some(attrs)) => {
+            Some(attrs
+                .iter()
+                .map(|field| schema
+                    .name(*field)
+                    .unwrap_or(&"unknown field name")
+                    .to_string())
+                .collect())
+        }
+        _ => None,
+    };
```

Comment:
I'm not sure if all this logic must be done at all, we should be able to retrieve an `HashSet` of the `attributes_for_faceting` and not check them. The update system must be correclty handling field removals and facets clean-ups.

Reply:
we store atrributes as `FieldId`, they must be converted back to strings so the user can read them.

Reply:
Are you sure this is `unwrap_or` that you want to use or `ok_or` or `expect` maybe?

Reply:
I don't want to panic, or return an error. I think I will just filter out values I can't find.

Reply:
truth is that it should not happen, or you have more worrying problem too